docs/how-to/authentication-plugin-examples.md

```diff
@@ -81,3 +81,37 @@ If you're using a .NET service, refer to the [.NET Auth Core Identity plugin doc
 3. Implement proper error handling and logging for authentication failures.
 4. Regularly update and rotate authentication secrets and keys.
 5. Follow the principle of least privilege when assigning roles and permissions.
+
+## Examples
+
+Here are some examples of how to use authentication in your service:
```

Comment:
Can we come up with more examples? 
The page title is "auth plugin examples", and it should be some kind of a guide- can we update it to be like that, i.e. with more "real" examples?